util/network-alert/src/alert_relayer.rs

```diff
@@ -33,9 +33,9 @@ pub struct AlertRelayer {
 }
 
 impl AlertRelayer {
-    pub fn new(client_version: String, config: Config) -> Self {
+    pub fn new(client_version: String, notifier_config: NotifierConfig, config: Config) -> Self {
```

Comment:
Why not merge them together?  Put `client_version` and `notify_script` into config when constructs. And `AlertRelayer` and `Notifier` could share the same config.